<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpriteRSP</class>
 <widget class="QMainWindow" name="SpriteRSP">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>265</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>385</width>
    <height>265</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>385</width>
    <height>265</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>가위바위보</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>-40</y>
      <width>181</width>
      <height>291</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>181</width>
      <height>291</height>
     </size>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>371</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>106</width>
         <height>40</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="minimumSize">
        <size>
         <width>105</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="minimumSize">
        <size>
         <width>106</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="winRate">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-40</y>
      <width>131</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Hack</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
